client/src/components/PreviousTripHashtagCard/PreviousTripHashtagCard.stories.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import PreviousTripHashtagCard from "./PreviousTripHashtagCard";
+
+storiesOf("PreviousTripHashtagCard", module)
+  .add("Previous Trips, No Data", () => (
+    <PreviousTripHashtagCard data={[]} type="trips">
+      <h1>Previous Trips</h1>
+      <h2>None with CoTripper yet!</h2>
+      <a href="#">Book a Trip Now!</a>
+    </PreviousTripHashtagCard>
+  ))
+  .add("Previous Trips", () => (
+    <PreviousTripHashtagCard
+      data={["New York", "Mexico", "Wakanda", "Hawaii", "Japan", "Canada", "Italy"]}
```

Comment:
One thing to note is that these hashtags might be links. So the data that would come through here as a prop might look more like:

```js
data={[{link: '/new-york', title: 'New York'},{link: '/mexico', title: 'Mexico'},{link: '/wakanda', title: 'Wakanda'},]}
```

Reply:
You can maybe find a way to handle both, or have this list be a `ul` that the user passes in as a child (like the text)